src/main/java/by/lawaksoft/tradebot/controller/trade/TradeController.java

```diff
@@ -0,0 +1,46 @@
+package by.lawaksoft.tradebot.controller.trade;
+
+import by.lawaksoft.tradebot.dto.amend_order.AmendOrderRequestDTO;
+import by.lawaksoft.tradebot.dto.cancel_order.CancelOrderRequestDTO;
+import by.lawaksoft.tradebot.dto.order.GetOrderDetailsDTO;
+import by.lawaksoft.tradebot.dto.order.GetOrderResponseDTO;
+import by.lawaksoft.tradebot.dto.place_order.PlaceOrderRequestDTO;
+import by.lawaksoft.tradebot.service.api.trade.ApiTradeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("${TRADE}")
```

Comment:
for controllers we can leave url path in String, because this App (inner) urls

external urls should be in properties, they can change

Reply:
done